jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/dao/RDSDBManager.java

```diff
@@ -39,7 +34,7 @@ public class RDSDBManager {
     private static final String DB_URL = System.getProperty("spring.datasource.url");
     private static final String DB_USER_NAME = System.getProperty("spring.datasource.username");
     private static final String DB_PASSWORD = System.getProperty("spring.datasource.password");
-
+    private static  final Set<String> excludeStatusUpdate = new HashSet<>(Arrays.asList("checkmarx"));//Paladin  Controlling status for Checkmarx
```

Comment:
This is also true for wiz, need to exclude both

Reply:
Modified approach as per latest sync policies design